libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/RustMatrixRoom.kt

```diff
@@ -113,6 +114,21 @@ class RustMatrixRoom(
         })
     }
 
+    override val roomTypingMembersFlow: Flow<List<UserId>> = mxCallbackFlow {
+        launch {
+            val initial = emptyList<UserId>()
+            channel.trySend(initial)
+        }
+        innerRoom.subscribeToTypingNotifications(object : TypingNotificationsListener {
+            override fun call(typingUsers: List<String>) {
+                channel.trySend(
+                    typingUsers
+                        .filter { it != sessionData.userId }
```

Comment:
Note that the SDK is also doing the filtering now, but it does not cost to do it twice.

Reply:
It's quite unlikely, but it may cause confusion in the future if the Rust SDK team assumes changing something on their side about this will be transparent for the clients, and then we have our own check here. I'm not saying it's wrong, but I'm not sure if it's more likely to cause problems than not having it at all.